v4l2loopback.c

```diff
@@ -1792,8 +1792,13 @@ static int vidioc_querybuf(struct file *file, void *fh, struct v4l2_buffer *buf)
 static void buffer_written(struct v4l2_loopback_device *dev,
 			   struct v4l2l_buffer *buf)
 {
-	del_timer_sync(&dev->sustain_timer);
-	del_timer_sync(&dev->timeout_timer);
+	#if LINUX_VERSION_CODE < KERNEL_VERSION(6, 15, 0)
+		del_timer_sync(&dev->sustain_timer);
+		del_timer_sync(&dev->timeout_timer);
+	#else
+		timer_delete_sync(&dev->sustain_timer);
+		timer_delete_sync(&dev->timeout_timer);
+	#endif
```

Comment:
i think it would be cleaner to just use the new API and set some compat macro(s) at the top of the file:
like https://github.com/v4l2loopback/v4l2loopback/blob/84cccedd9d3979f0a8ec5478b100ea57adf29696/v4l2loopback.c#L44-L46